src/briefcase/integrations/cookiecutter.py

```diff
@@ -139,6 +139,11 @@ def bool_attr(obj):
             """Render value in XML format appropriate for an attribute."""
             return "true" if obj else "false"
 
+        def escape_xml(obj):
+            """Escapes single quotes and backslashes."""
+            return obj.replace("\\", "\\\\").replace("'", "\\'")
```

Comment:
May be best way to escape `'` and `"` is using `"'"` or `'"'`.
>To allow attribute values to contain both single and double quotes, the apostrophe or single-quote character (') may be represented as " &apos; ", and the double-quote character (") as " &quot; ".
See https://www.w3.org/TR/REC-xml/#syntax

Reply:
So best thing to do here might be replacing `'` with `&apos;`.

Reply:
Also https://docs.python.org/3/library/xml.sax.utils.html#xml.sax.saxutils.quoteattr might be of help.